### PR TITLE
Unify node types

### PR DESCRIPTION
**Pull request by bendaamerahm**
unify nodes types:

- fusion of selectedNode and workingNode
- remove eye icon from selected node

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6526,7 +6526,7 @@
         "diacritic": {
             "version": "0.0.2",
             "resolved": "https://registry.npmjs.org/diacritic/-/diacritic-0.0.2.tgz",
-            "integrity": "sha512-iQCeDkSPwkfwWPr+HZZ49WRrM2FSI9097Q9w7agyRCdLcF9Eh2Ek0sHKcmMWx2oZVBjRBE/sziGFjZu0uf1Jbg=="
+            "integrity": "sha1-/CqIe1pbwKCoVPthTHwvIJBh7gQ="
```
Comment (JacquesBorsenberger) [inline package-lock.json:6529]: is this change wanted ?

**Review by JacquesBorsenberger — APPROVED** [review]

File: src/components/graph/util/model-functions.js [hunk 2]
```diff
@@ -30,29 +30,9 @@ export function getFirstNodeOfType(elements, nodeType, buildStatus) {
...
 
-export function isNodeValid(workingNode, selectedNode) {
-    const readOnlyNode = workingNode?.readOnly;
-    const builtWorkingNode = workingNode?.buildStatus === 'BUILT';
-    const workingNodeIsSelectedNode =
-        selectedNode === null ||
-        selectedNode === undefined ||
-        workingNode?.id === selectedNode?.id;
+export function isNodeValid(currentNode) {
+    const readOnlyNode = currentNode?.data?.readOnly;
```
Comment (JacquesBorsenberger) [inline src/components/graph/util/model-functions.js:34]: is a read only node always valid or is this true for root ?

File: package-lock.json (hunk 1 above)
Comment (bendaamerahm) [reply package-lock.json:6529]: i don't know how this is happened

File: src/components/graph/util/model-functions.js (hunk 2 above)
Comment (SlimaneAmar) [reply src/components/graph/util/model-functions.js:34]: Why readOnly ?
just test type != ROOT ?

Comment (SlimaneAmar) [reply src/components/graph/util/model-functions.js:34]: Just  `currentNode?.data?.type == ROOT || currentNode?.data?.buildStatus === 'BUILT'`

**Review by sBouzols — APPROVED** [review]
Code Review OK
Tests Done :
* Tree navigation
* Topbar node name update
* Loadflow
* Network Modifacation Creation/edition/Deletion ...
* Log by node
* spreadsheet by node
* node realization
Console Warning Checked

File: src/components/graph/util/model-functions.js (hunk 2 above)
Comment (bendaamerahm) [reply src/components/graph/util/model-functions.js:34]: good question this check is for all nodes

**Pull request merged**